.github/workflows/gen_windows_tag.yml

```diff
@@ -20,12 +20,8 @@ jobs:
         with:
           target: "x86_64-pc-windows-msvc"
       - name: "Cache cargo"
-        uses: actions/cache@v3
+        uses: Swatinem/rust-cache@v2
         with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            target
           key: "windows-x86_64-pc-windows-msvc-2-${{ runner.os }}-${{ hashFiles('**/Cargo.lock') }}-cargo"
```

Comment:
it may be worth revisiting the `hashFiles('**/Cargo.lock')` portion of this.  Right now any change to the deps in the Cargo.lock will invalidate the whole cache, but it feels desirable to reuse some of the previous cache if only one or two deps are different

Reply:
I should remove it but the action itself hashes all Cargo.lock/Cargo.toml files https://github.com/Swatinem/rust-cache#cache-details so it makes little difference

Reply:
Then removing just makes things simpler on this end, so that's fine :)

Reply:
Solved in latest commit!